src/Microsoft.Sbom.Api/Output/MetadataBuilder.cs

```diff
@@ -42,6 +42,14 @@ public MetadataBuilder(
     /// <returns></returns>
     public string GetHeaderJsonString(IInternalMetadataProvider internalMetadataProvider)
     {
+        // SPDX 3.0 and above handles writing the info in metadata dictionary differently.
```

Comment:
We don't know what versions after 3 will do
```suggestion
        // SPDX 3.0 handles writing the info in metadata dictionary differently.
```